src/modals/import-account/import-account-seed/import-account-from-seed.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { createUseStyles } from 'src/styles/create-use-styles';
+import { formatSize } from 'src/styles/format-size';
+
+export const useImportAccountFromSeedStyles = createUseStyles(() => ({
+  seedPhraseInputContainer: {
+    flexGrow: 1
+  },
+  buttonsContainer: {
+    paddingHorizontal: formatSize(8)
+  },
+  buttonBox: {
```

Comment:
Please consider using `flex: 1` and a divider in such cases. Example is here: https://github.com/madfish-solutions/templewallet-mobile/blob/development/src/components/earn-opportunity-item/index.tsx#L191